app/stac_api/serializers.py

```diff
@@ -32,6 +32,46 @@ def create_or_update_str(created):
     return 'update'
 
 
+def update_or_create_links(model, instance, instance_type, links_data):
```

Comment:
maybe add some comment about what the parameters are, what kind of values you expect